addons/point_of_sale/static/tests/tours/utils/product_screen_util.js

```diff
@@ -1,6 +1,6 @@
 import * as Numpad from "@point_of_sale/../tests/tours/utils/numpad_util";
 import * as Order from "@point_of_sale/../tests/tours/utils/generic_components/order_widget_util";
-import { inLeftSide } from "@point_of_sale/../tests/tours/utils/common";
+import { back as utilsBack, inLeftSide } from "@point_of_sale/../tests/tours/utils/common";
```

Comment:
why do you need to alias the back method ?

Reply:
in that file i have 
```js
export function back() {
    return utilsBack();
}
```
The `back` function is actually defined in `utils/common.js`, but many places are calling `ProductScreen.back()`